ribasim_qgis/tests/core/test_model.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+from qgis.testing import unittest
+
+from ribasim_qgis.core.model import (
+    get_database_path_from_model_file,
+    get_directory_path_from_model_file,
+)
+
+
+class TestModel(unittest.TestCase):
+    tests_folder_path = Path(__file__).parent.parent.resolve()
+    data_folder_path = tests_folder_path / "data"
+
+    def test_get_directory_path_from_model_file(self):
+        """_summary
+        Tests that get_directory_path_from_model_file() can resolve paths from a toml file.
+        """
```

Comment:
```suggestion
        """Tests that get_directory_path_from_model_file() can resolve paths from a toml file."""
```
We try to follow numpydoc style: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Without the `summary` its compliant I would say